docs/views/templates/blank-transactional.html

```diff
@@ -9,11 +9,10 @@
 {% block beforeContent %}
 {% endblock %}
```

Comment:
Remove these

Reply:
Opps, well spotted!

Reply:
Fixed in 2babbad60a0aa35d2a5e085cf72280487025622e